app/src/main/java/study_dev/testbed/studyhopper/ui/dashboard/DashboardFragment.java

```diff
@@ -113,11 +113,14 @@ public void onComplete(@NonNull Task<QuerySnapshot> task) {
                                     @Override
                                     public void onItemClick(DocumentSnapshot documentSnapshot, int position) {
                                         String id = documentSnapshot.getId();
+                                        Group mockGroup = documentSnapshot.toObject(Group.class);
+                                        String groupDocId = mockGroup.getDocumentId().getId();
                                         Toast.makeText(getContext(),
                                                 "Position: " + position + " ID: " + id, Toast.LENGTH_SHORT).show();
                                         Activity activity = (Activity) getContext();
                                         Intent intent = new Intent(getContext(), StudyGroupActivity.class);
                                         intent.putExtra("documentID", id);
+                                        intent.putExtra("groupDocId", groupDocId);
```

Comment:
I would rename these to make it clearer which is which.

Reply:
Adjusted names to make it clearer.